contracts/tg4-engagement/Cargo.toml

```
@@ -20,21 +20,21 @@ backtraces = ["cosmwasm-std/backtraces"]
 library = []
 
 [dependencies]
-cw-utils = "0.13.1"
-cw2 = "0.13.1"
-cw-controllers = "0.13.1"
-cw-storage-plus = "0.13.1"
-tg4 = { path = "../../packages/tg4", version = "0.9.0" }
-tg-utils = { version = "0.9.0", path = "../../packages/utils" }
-tg-bindings = { version = "0.9.0", path = "../../packages/bindings" }
-cosmwasm-std = { version = "1.0.0-beta8" }
+cw-utils = "0.13.4"
+cw2 = "0.13.4"
+cw-controllers = "0.13.4"
+cw-storage-plus = "0.13.4"
+tg4 = { path = "../../packages/tg4", version = "0.10.0" }
+tg-utils = { version = "0.10.0", path = "../../packages/utils" }
+tg-bindings = { version = "0.10.0", path = "../../packages/bindings" }
+cosmwasm-std = { version = "1.0.0" }
```

```suggestion
cosmwasm-std = "1.0.0"
```
In couple more places.
Every time I think we got rid of that syntax it comes back again somehow. :)